dont-bother-looking.md

```diff
@@ -0,0 +1,49 @@
+## Title
+* Don't bother looking
+
+## Also Known As
+* Not looking for stuff internally 
+
+## Context
+* Software component(s) are available internally but users can't easily find these.
+* This problem is more likely to occur where there are silos in the company (e.g., larger companies; smaller companies may have fewer opportunities for reuse of internally developed software). 
+* The company traditionally has been bad at sharing across silos (people don't have the culture of sharing). 
+
+## Problem
+People don't bother looking for internally developed solutions - they might not find the repo at all or be aware of its existence.
+
+## Forces
+* No good internal search engine (or not connected to git repositories; and difficult to make this change happen)
+* Users may not know there are common places to find internally developed solutions.
+* People don't expect to find solutions internally.
+* Many silos in the company; difficult to reach the developer base across those silos (a communications problem).
+* People might not want to use internal SW because they don't believe it will be helpful
+    - might not be maintained
+    - might have poor reusability
+    - if someone put out a SW internally, the expectation is that they wouldn't have time to support it (vs. open source options)
+
+## Solution
+Make it easy to find the reusable code.
```

Comment:
Another avenue to explore is making it easy for the reusable code to find you.  Create ways for the people running shared internal code to discover which other projects are reasonable candidates for adoption so that they can reach out.

Reply:
Do you have some examples of what works to solve the discovery problem?

Reply:
Search through source code at the company for projects that are candidates to use the shared internal tool and then reach out to committers to those repos.

Reply:
* Encourage (and reward) owners of reusable code to use the same search engine to continually search for products that are candidates for use and adoption of the reusable code but not currently doing so.